TELCODOCS-1868: Moving feature host network settings for SR-IOV network VFs to GA with minor functionality updates (#80046)
openshift/openshift-docs: modules/virt-example-vf-host-services.adoc

<9> Identifies the VF with an ID of `0`.
<10> Sets a maximum transmission rate, in Mbps, for the VF. This sample value sets a rate of 200 Mbps.

> The following YAML file is an example of a manifest that adds a VF for a network interface. In this configuration, the `ens1f1v0` VF is created on the `ens1f1` physical interface, and this VF is added to a bonded network interface `bond0`. The bond uses `active-backup` mode for redundancy. In this example, the VF is configured to use hardware offloading to manage the VLAN directly on the physical interface.

Reviewer: Another way of doing this to break up the information that is a bit dense is to move everything after first sentence to after the code block as an explanation of it. My next comment has an alternative method as well.

Author: Thanks for your points on this one Joe. I've added a line break after the first sentence. And I removed the line that says to replace with your own values. I also "In this sample configuration..." now. 

The callouts describe the fields, rather than the relationship between the fields in this particular example. As there's quite a lot of inter-dependent networking going on here, I think a single para describing it as a whole is worthwhile.

> This YAML includes samples values that you must replace with your own information.

Reviewer: It isn't clear what these values are because there aren't any <replace_with_your_value> in the example. I recommend actually moving a lot of this to comments in the YAML file.  For example, the first second sentence on line 59 could be moved to commented out material on line 74 and 84.

  maxUnavailable: 3
  desiredState:
    interfaces:
      - name: ens1f1 <4>
        type: ethernet
        state: up
        ethernet:
            sr-iov:
              total-vfs: 1 <5>
              vfs:
                - id: 0
                  trust: true <6>
                  vlan-id: 477 <7>
      - name: bond0 <8>
        description: Attach VFs to bond <9>
        type: bond <10>
        state: up <11>
        link-aggregation:
          mode: active-backup <12>
          options:
            primary: ens1f0v0 <13>
          port: <14>
            - ens1f0v0
            - ens1f1v0 <15>
----
<1> Name of the policy.
<2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.

> <3> This example applies to all nodes with the `worker` role.

Reviewer: ```suggestion
<3> The example applies to all nodes with the `worker` role.
```

<4> Name of the VF network interface.
<5> Number of VFs to create.
<6> Setting to allow failover bonding between the active and backup VFs.

> <7> ID of the VLAN. This example uses hardward offloading to define a VLAN directly on the VF.

Reviewer: ```suggestion
<7> ID of the VLAN. The example uses hardware offloading to define a VLAN directly on the VF.
```

<8> Name of the bonding network interface.
<9> Optional: Human-readable description of the interface.
<10> The type of interface.
<11> The requested state for the interface after configuration.
<12> The bonding policy for the bond.
<13> The primary attached bonding port.
<14> The ports for the bonded network interface.

> <15> In this example, this VLAN network interface is added as an additional interface to the bonded network interface.

Reviewer: ```suggestion
<15> In this example, the VLAN network interface is added as an additional interface to the bonded network interface.
```
